hnn_core/tests/test_extracellular.py

```diff
@@ -0,0 +1,341 @@
+# Authors: Nick Tolley <nicholas_tolley@brown.edu>
+#          Christopher Bailey <cjb@cfin.au.dk>
+
+from copy import deepcopy
+import os.path as op
+import numpy as np
+from numpy.linalg import norm
+from numpy.testing import assert_allclose, assert_array_equal
+import pytest
+
+import hnn_core
+from hnn_core import read_params, default_network, simulate_dipole, Network
+from hnn_core.extracellular import ExtracellularArray
+from hnn_core.parallel_backends import requires_mpi4py, requires_psutil
+from hnn_core.parallel_backends import MPIBackend
+
+
+def test_extracellular_api():
+    """Test extracellular recording API."""
+    hnn_core_root = op.dirname(hnn_core.__file__)
+    params_fname = op.join(hnn_core_root, 'param', 'default.json')
+    params = read_params(params_fname)
+    net = default_network(deepcopy(params), add_drives_from_params=True)
+
+    # Test LFP electrodes
+    electrode_pos = (2, 400, 2)
```

Comment:
is this still with the xy axis swap? Probably doesn't matter for test but might be worth switching

Reply:
throw a warning/error if all electrodes are too far from any cell?

Reply:
Didn't see that! That test function just tests the API, but I still made some clarifications to emphasise what's being tested.

Not sure there's a heuristic for "too far", especially since we want to compare with "macroscopic" recordings.